### PR TITLE
refactor: byzer taxi demo

### PR DESCRIPTION
**Pull request by vagetablechicken**

### CODE REVIEW

File: docs/zh/use_case/OpenMLDB_Byzer_taxi.md [hunk 1]
```diff
@@ -166,73 +219,121 @@ and `fitParam.0.maxIter`="50";
...
+SET @@sync_job=true;
 '''
 and `sql-2`='''
-CREATE TABLE t1(id string, vendor_id int, pickup_datetime timestamp, dropoff_datetime timestamp, passenger_count int, pickup_longitude double, pickup_latitude double, dropoff_longitude double, dropoff_latitude double, store_and_fwd_flag string, trip_duration int);
+SET @@job_timeout=20000000;
 '''
 and `sql-3`='''
-LOAD DATA INFILE 'tmp/upload/test.csv'
-INTO TABLE t1 options(format='csv',header=true,mode='append');
+LOAD DATA INFILE '/byzermnt/tmp/upload/taxi_tour_table_train_simple.csv'
```
Comment (vagetablechicken) [inline docs/zh/use_case/OpenMLDB_Byzer_taxi.md:272]: en doc

File: docs/zh/use_case/OpenMLDB_Byzer_taxi.md [hunk 2]
```diff
@@ -6,113 +6,171 @@
...
-2. 本例中，Byzer引擎需要从容器外部访问OpenMLDB服务，需要修改OpenMLDB的原始IP配置，修改方式详见[IP配置文档](../reference/ip_tips.md)。
+1. 本例推荐使用运行在Docker容器中的OpenMLDB集群版。安装步骤详见[OpenMLDB快速上手](../quickstart/openmldb_quickstart.md)。
+1. 本例中，Byzer引擎虽然在同一主机上，但需要从容器外部访问OpenMLDB服务，所以需要暴露OpenMLDB集群的服务端口，推荐使用`--network host`的方式，具体详见[IP配置文档-CLI/SDK->容器onebox](../reference/ip_tips.md#clisdk-容器onebox)。
+1. 为了简单起见，我们使用file形式导入导出OpenMLDB集群数据，所以需要Byzer和OpenMLDB共享文件路径，这里我们将`/mlsql/admin`映射为`/byzermnt`，与OpenMLDB交互的SQL命令中使用`/byzermnt`开头作为文件路径。
+1. 我们还需要在OpenMLDB集群中创建名为`db1`的数据库，接下来在Byzer中使用这一数据库（目前不能在Byzer中执行，Byzer连接OpenMLDB必须指定可使用的db）。
 
-### 1.2 安装 Byzer 引擎和Byzer Notebook 
+执行命令如下：
+```
+docker run --network host -dit --name openmldb -v /mlsql/admin/:/byzermnt 4pdosc/openmldb:0.6.3 bash
```
Comment (dl239) [inline docs/zh/use_case/OpenMLDB_Byzer_taxi.md]: 0.6.4

Comment (vagetablechicken) [reply docs/zh/use_case/OpenMLDB_Byzer_taxi.md]: only update 064 in this doc file

File: docs/zh/reference/ip_tips.md [hunk 3]
```diff
@@ -129,7 +129,10 @@ docker run -p 6527:6527 -p 9921:9921 -p 8080:8080 -it 4pdosc/openmldb:0.6.3 bash
 docker run -p 2181:2181 -p 7527:7527 -p 10921:10921 -p 10922:10922 -p 8080:8080 -p 9902:9902 -it 4pdosc/openmldb:0.6.3 bash
 ```
 
-- 使用host网络，可以不用修改endpoint配置。见[容器onebox-apiserver](#容器onebox-apiserver)
+- 使用host网络，可以不用修改endpoint配置。如果有端口冲突，请修改server的端口配置。
+```
+docker run --network host -it 4pdosc/openmldb:0.6.3 bash
```
Comment (dl239) [inline docs/zh/reference/ip_tips.md]: 0.6.4

File: docs/zh/use_case/OpenMLDB_Byzer_taxi.md [hunk 4]
```diff
@@ -6,113 +6,171 @@
...
-2. 本例中，Byzer引擎需要从容器外部访问OpenMLDB服务，需要修改OpenMLDB的原始IP配置，修改方式详见[IP配置文档](../reference/ip_tips.md)。
+1. 本例推荐使用运行在Docker容器中的OpenMLDB集群版。安装步骤详见[OpenMLDB快速上手](../quickstart/openmldb_quickstart.md)。
+1. 本例中，Byzer引擎虽然在同一主机上，但需要从容器外部访问OpenMLDB服务，所以需要暴露OpenMLDB集群的服务端口，推荐使用`--network host`的方式，具体详见[IP配置文档-CLI/SDK->容器onebox](../reference/ip_tips.md#clisdk-容器onebox)。
+1. 为了简单起见，我们使用file形式导入导出OpenMLDB集群数据，所以需要Byzer和OpenMLDB共享文件路径，这里我们将`/mlsql/admin`映射为`/byzermnt`，与OpenMLDB交互的SQL命令中使用`/byzermnt`开头作为文件路径。
+1. 我们还需要在OpenMLDB集群中创建名为`db1`的数据库，接下来在Byzer中使用这一数据库（目前不能在Byzer中执行，Byzer连接OpenMLDB必须指定可使用的db）。
 
-### 1.2 安装 Byzer 引擎和Byzer Notebook 
+执行命令如下：
+```
+docker run --network host -dit --name openmldb -v /mlsql/admin/:/byzermnt 4pdosc/openmldb:0.6.4 bash
```
Comment (dl239) [inline docs/zh/use_case/OpenMLDB_Byzer_taxi.md:16]: add file to https://github.com/4paradigm/OpenMLDB/blob/main/steps/upgrade_docs_version.sh#L92

**Review by dl239 — APPROVED** [review]

File: docs/zh/reference/ip_tips.md (hunk 3 above)
Comment (vagetablechicken) [reply docs/zh/reference/ip_tips.md]: I'll merge main

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**